src/Client/Web/Bit.Client.Web.BlazorUI/Components/Rating/BitRating.razor.cs

```diff
@@ -24,13 +26,27 @@ public bool IsReadonly
             }
         }
 
+        [Parameter]
+        public int Value
+        {
+            get => ratingValue;
+            set
+            {
+                ratingValue = value;
```

Comment:
please check if the value is not actually changed the method returns fast.
just like the BitCheckBox component

Reply:
this is not fixed.
its required to check if the value is actually changed:

```cs
set
{
    if (value == ratingValue) return;
```